modules/nw-creating-a-route.adoc

```diff
@@ -0,0 +1,73 @@
+[id="creating-a-route"]
+= Creating a route
+
+
+
+A route allows you to host your application at a public URL. It can either be secure or insecure depending on the network security configuration of your application. An insecure route uses the basic HTTP routing protocol and exposes a service on an insecure application port.
```

Comment:
Add a comma- It can either be secure or insecure, depending on the network...

Reply:
Done. Thanks for the review.